packages/front-end/components/Features/CopyRuleModal.tsx

```diff
@@ -32,13 +32,29 @@ export default function CopyRuleModal({
   const filteredEnvironments = environments.filter(
     (env) => env.id !== environment
   );
+  const envs = filteredEnvironments.map((e) => e.id);
 
   const ruleTxt = rules.length === 1 ? "rule" : "rules";
 
   const [selectedEnvironments, setSelectedEnvironments] = useState<
     Record<string, boolean>
   >({});
 
+  const allToggled =
+    Object.values(selectedEnvironments).filter((v) => v).length === envs.length;
```

Comment:
the filter doesn't look like it is doing anything here i think we can remove that
```suggestion
    const allToggled = Object.values(selectedEnvironments).length === envs.length;
```

Reply:
wait do we even need object.values here?

Reply:
nah it's needed, otherwise it includes items like `prod: false`

Reply:
wait it does its a boolean value :)